HACKING.md

```diff
@@ -42,97 +42,60 @@ oc scale --replicas=0 deployment.apps/local-storage-operator -n openshift-local-
 All the images including operator, diskmaker, bundle and index images can be created in one shot using following command:
 
 ```
-~> ./hack/sync_bundle -o <operator_image> -d <diskmaker_image> -b <bundle_image> -i <index_image> bundle
+make bundle REGISTRY=quay.io/username
 ```
 
-This command also pushes the images to selected docker registry, so if you ran this command with following arguments:
+You may optionally set `VERSION` to be used as a tag, and `TOOL_BIN` to specify which tool to use (defaults to podman or docker).
```

Comment:
Imagine that the user has both podman and docker installed, but he/she wants to use docker as in your example below:
```
make bundle REGISTRY=quay.io/username VERSION=latest TOOL_BIN=`which docker`
```
Then docker will be used only in Makefile, but [`hack/create-bundle`](https://github.com/openshift/local-storage-operator/pull/433/commits/8aa1d0729e319e4bddef7d9ea68bcf9dd738f4a9#diff-d9ff92edcb92263c6a6d63ab25bcd1abb0ad4825f599d479d31636840b118eb8R15) would still use podman.

I think it's not nice. We have to pass env var `TOOL_BIN` to the script and use it (if it is set).

Reply:
Yes good catch! I addressed this in the last commit.